config/rapidez/frontend.php

```diff
@@ -89,4 +89,11 @@
 
     // The path to redirect to after a failed checkout
     'checkout_success_fail_redirect_path' => 'cart',
+
+    'filters' => [
+        // Show inputs below a range slider (true/false)
+        'range_slider' => [
+            'inputs' => true,
+        ],
+    ],
```

Comment:
This can be removed, just keep it `true` within the component prop. If we want this configurable someday we'll handle it differently.